kythe/cxx/indexer/cxx/testdata/rec/rec_implicit_ctor_calls_conversion.cc

```diff
@@ -0,0 +1,19 @@
+// ref/call anchors should cover the expression triggering the conversion.
+struct T {
+//- @T defines/binding TCharCtor
+  T(const char*) {}
+//- @T defines/binding TIntCtor
+  T(int) {}
+};
+void c(T);
+void fn() {
+  // The anchor for implicit conversions should span the entire expression.
+  //- String ref/call TCharCtor
+  //- String.loc/start @$"("
+  //- String.loc/end @$z
```

Comment:
Probably ")"?

Reply:
It should be quotation mark to quotation mark, but the simplest way of spelling that seemed to be:
"after the preceding parenthesis" (`@$"("`) and "after the last character" (`@$z`)

Reply:
Fixed.